contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -39,29 +35,14 @@ contract UniswapAnchoredView is AggregatorValidatorInterface, UniswapConfig, Own
     /// @notice Official prices by symbol hash
     mapping(bytes32 => PriceData) public prices;
 
-    /// @notice The old observation for each symbolHash
-    mapping(bytes32 => Observation) public oldObservations;
-
-    /// @notice The new observation for each symbolHash
-    mapping(bytes32 => Observation) public newObservations;
-
     /// @notice The event emitted when new prices are posted but the stored price is not updated due to the anchor
     event PriceGuarded(bytes32 indexed symbolHash, uint reporter, uint anchor);
 
     /// @notice The event emitted when the stored price is updated
     event PriceUpdated(bytes32 indexed symbolHash, uint price);
 
-    /// @notice The event emitted when anchor price is updated
-    event AnchorPriceUpdated(bytes32 indexed symbolHash, uint anchorPrice, uint oldTimestamp, uint newTimestamp);
-
-    /// @notice The event emitted when the uniswap window changes
-    event UniswapWindowUpdated(bytes32 indexed symbolHash, uint oldTimestamp, uint newTimestamp, uint oldPrice, uint newPrice);
-
     /// @notice The event emitted when failover is activated
-    event FailoverActivated(bytes32 indexed symbolHash);
-
-    /// @notice The event emitted when failover is deactivated
-    event FailoverDeactivated(bytes32 indexed symbolHash);
+    event FailoverUpdated(bytes32 indexed symbolHash, bool status);
```

Comment:
I agree - this was already here from the Kovan code. Re-adding the separate events now bumps the bytecode size to 24.14KB. How important do you think this would be vs maximising the number of token configs?